examples/diff.c

```diff
@@ -335,3 +363,27 @@ static void diff_print_stats(git_diff *diff, struct opts *o)
 	git_buf_free(&b);
 	git_diff_stats_free(stats);
 }
+
+int extract_break_rewrites(int *m, int *n, const char *arg)
+{
```

Comment:
I'm just going through the things that @tiennou has pointed out one by one.  The next one that jumps out at me is that `m_str` and `n_str` are unused.